test/_test_data/copyright_file_contents/test_pkg_spdx_name

```diff
@@ -5,7 +5,7 @@ Upstream-Name: test_pkg_spdx_name
 Files:
  **
 Copyright: 2003 Lawrence E. Rosen
-License: AFL-2.0
+License: Academic Free License v2.0
```

Comment:
Why is it the name here? Where is this resolved?

Reply:
Because we now cannot translate from long version (Academic Free License v2.0) to short (AFL-2.0). But at second thought it might make more sense to adapt the tag in the module, wouldn't it?